opensrp-reveal/src/main/java/org/smartregister/reveal/presenter/TaskRegisterFragmentPresenter.java

```diff
@@ -62,6 +68,11 @@
 
     private boolean isActionClicked = true;
 
+    private TaskFilterParams filterParams;
+
+    private boolean isTasksFiltered;
+
+    private ArrayList<TaskDetails> filteredTasks;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'filteredTasks' could be replaced by a local variable.](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=4684568)